HOMEapp/Brandhome/Manifest/manifest.json

```diff
@@ -8,7 +8,8 @@
     "name": "Microsoft",
     "websiteUrl": "https://www.microsoft.com",
     "privacyUrl": "https://www.microsoft.com/privacy",
-    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse",
+    "mpnId": "{your mpnID}"
   },
```

Comment:
Add a comment here, <-- Microsoft Partners must include their MPNID in the App Manifest when they build the App. It is being used for telemetry and usage reporting purposes -->

Reply:
Updated